kubetest/anywhere.go

```diff
@@ -32,6 +32,7 @@ import (
 )
 
 const defaultKubeadmCNI = "weave"
+const defaultBuildFolder = "bazel"
```

Comment:
only PR builds still go here. All CI bazel builds are `gs://kubernetes-release-dev/ci/%v-bazel` and we should get this path from `gs://kubernetes-release-dev/ci/latest-bazel.txt` for CI. I'm working on a patch for this in the various places that specify it.

Reply:
I figured we could use the flag for a controlled switch over and then remove the flag.

Reply:
these jobs are all failing right now, we might as well just switch them over. I'm trying to fix CI and then sort out the presubmits. see https://github.com/kubernetes/test-infra/pull/7139

Reply:
Fair enough. Will close this out in favor of #7139